docs/sites.yml

```diff
@@ -3723,3 +3723,14 @@
     - Business
     - Food
     - Beverage
+- title: Satispay
+  url: https://www.satispay.com
+  main_url: https://www.satispay.com
+  categories:
+    - Business
+    - Finance
+    - Fintech
```

Comment:
Please remove this tag, as it's a duplication of `Finance`

```suggestion
```

Reply:
Done 👍